libsql-server/src/connection/connection_core.rs

```diff
@@ -88,18 +87,6 @@ impl<W: Wal + Send + 'static> CoreConnection<W> {
             config.max_row_size as i32,
         );
 
-        unsafe {
```

Comment:
What's the effect of this being removed?

Reply:
We will have default busy handler which will abort after busy timeout and internally sleep and wake up with some intervals